build.ps1

```diff
@@ -0,0 +1,105 @@
+param (
```

Comment:
I'm sorry, but I would rather not maintain two different build scripts.
I'm also on Windows and just use WSL to build the plugin